### PR TITLE
notification

### PR DESCRIPTION
**Pull request by Bartal007**
I add notification with progress bar. I hope this time it will be OK.

### CODE REVIEW

File: app/src/main/java/eu/quelltext/mundraub/activities/SettingsActivity.java [hunk 1]
```diff
@@ -195,6 +206,50 @@ public void onClick(View v) {
...
+
+        NotificationManager notificationManager= (NotificationManager) getSystemService(NOTIFICATION_SERVICE);
+        notificationManager.notify(uniqueID, notification.build());
+
+        Thread thread = new Thread(){
+            @Override
+            public void run() {
+                int count=0;
+                try {
+                    while (count <= 100) {
```
Comment (niccokunzmann) [inline app/src/main/java/eu/quelltext/mundraub/activities/SettingsActivity.java]: Thanks for showing this code. This is probably a good prototype for how the progress can be implemented. #147 
For this to go live, I would expect it to display the actual progress made. 
I think, you can put code into ProgressUpdate https://github.com/niccokunzmann/mundraub-android/blob/b3c97c078d8eefdc34653f00eea5601520194473/app/src/main/java/eu/quelltext/mundraub/activities/SettingsActivity.java#L65 
Putting it there allows updating the progress bar in a uniform way.

**Comment by niccokunzmann** [comment]
I can merge it. What are your thoughts?

**Comment by Bartal007** [comment]
That would be great if you merge it, I would be happy that I helped a little bit. :) Maybe in the next few weeks if I'll have a time I can look on it.

**Comment by niccokunzmann** [comment]
Okay. I merge it and refine the issue.

**Pull request merged**